include/tvm/node/reflection.h

```diff
@@ -404,5 +404,11 @@ inline bool ReflectionVTable::GetReprBytes(const Object* self, std::string* repr
   }
 }
 
+/*!
+ * \brief Given an object and an address of its attribute, return the key of the attribute.
+ * \return nullptr if no attribute with the given address exists.
+ */
+const char* GetAttrKeyByAddress(const Object* object, const void* attr_address);
```

Comment:
nit: it's adding a bit of overhead, but let's use `std::string` or `tvm::runtime::String` when possible

```suggestion
String GetAttrKeyByAddress(const Object* object, const void* attr_address);
```